src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -349,7 +349,13 @@ def preprocess(self, inputs, chunk_length_s=0, stride_length_s=None, ignore_warn
             inputs = _inputs
             if in_sampling_rate != self.feature_extractor.sampling_rate:
                 import torch
-                from torchaudio import functional as F
+
+                if is_torchaudio_available():
+                    from torchaudio import functional as F
+                else:
+                    raise ImportError(
+                        "torchaudio is required to resample audio samples in AutomaticSpeechRecognitionPipeline."
```

Comment:
Maybe add `pip install torchaudio`?

Reply:
Resolved in https://github.com/huggingface/transformers/pull/23953/commits/64e6365d115e9c70e683362aa68757a0d32bc295